assets/css/style.css

```diff
@@ -0,0 +1,323 @@
+body {
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+
+    font-size: 15px;
+    line-height: 1.6;
+    background-color: #576A7D;
+}
+
+*,
+* :after,
+* :before {
+    box-sizing : border-box
+}
+
+h1, h2, h3, h4, h5, h6 {
+    margin: 0;
+}
+
+.container {
+    width: 100%;
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+.intro {
+    width: 100%;
+    height: 100vh;

```

Comment:
it's not good to use vh especially for mobile phones. for making full screen size element it's better to use % for now.